android/src/main/java/com/tailscale/ipn/ui/model/TailCfg.kt

```diff
@@ -107,13 +107,9 @@ class Tailcfg {
     val isExitNode: Boolean =
         (AllowedIPs?.contains("0.0.0.0/0") ?: false) && (AllowedIPs?.contains("::/0") ?: false)
 
-    // mullvad nodes are exit nodes with a mullvad.ts.net domain *or* Location Info.
-    // These checks are intentionally redundant to avoid false negatives.
+    // mullvad nodes are exit nodes with non-nil Location info.
```

Comment:
This isn't a future-proof heuristic. 

We plan to add Location fields to other types of nodes.